File: web/js/Avion.js
 // funcion Avion
function Avion (identificador,anno,modelo,marca,cantPas,cantFilas,cantAsientos){ 
  this.Avion(identificador,anno,modelo,marca,cantPas,cantFilas,cantAsientos);
}

// prototype asociado a la funcion Persona
// solo metodos
Avion.prototype={ 
  Avion: function(identificador,anno,modelo,marca,cantPas,cantFilas,cantAsientos){
    this.identificador=identificador;
    this.anno=anno;
	this.modelo=modelo;
	this.marca=marca;
	this.cantPas=cantPas;
	this.cantFilas=cantFilas;
	this.cantAsientos=cantAsientos;
  },
  completo: function (sep) { return this.identificador; }
}

File: src/java/airline/model/Avion.java
package airline.model;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author John
 */
public class Avion implements Jsonable{
    String identificador;
    String anno;
    String modelo;
    String marca;
    int cantPas;
    int cantFilas;
    int cantAsientos;
    
    public Avion( String id, String anno, String modelo, String marca, int cantPas, int cantFilas, int cantAsientos){
        this.identificador=id;
        this.anno=anno;
	this.modelo=modelo;
	this.marca=marca;
	this.cantPas=cantPas;
	this.cantFilas=cantFilas;
	this.cantAsientos=cantAsientos;
    
    }
}
